sdk/helper/ldaputil/client.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/hashicorp/go-secure-stdlib/tlsutil"
 )
 
+// Deprecated: Use ldap.Client instead
```

Comment:
I'm thinking we'll want to leave this as not deprecated. It's still used in a couple of different plugins ([openldap example](https://github.com/hashicorp/vault-plugin-secrets-openldap/blob/main/client/client.go#L41)) for secrets engine use cases. I don't think cap/ldap will ever replace the usage given that it's targeted at authentication.